### ISSUE TITLE
`FM_GATEWAY_FEES` environment variable appears to be somewhat misleading.

### ISSUE THREAD

**Issue by hensg** [fedimint/fedimint#4915]
The `FM_GATEWAY_FEES` environment variable appears to be somewhat misleading.

It is utilized only during the initial setup to establish gateway fees. However, if the fees have already been configured or subsequently updated, the variable has no effect, and the fees are instead retrieved from the database.

**Comment by dpc** [comment]
Should we rename `FM_INIITIAL_GATEWAY_FEES` or `FM_DEFAULT_GATEWAY_FEES`?

**Comment by m1sterc001guy** [comment]
> Should we rename FM_INIITIAL_GATEWAY_FEES or FM_DEFAULT_GATEWAY_FEES?

Yes. This was recently changed, apparently we just didn't update the name of this constant.

**Comment by dpc** [comment]
> Should we rename `FM_INIITIAL_GATEWAY_FEES` or `FM_DEFAULT_GATEWAY_FEES`?

BTW. In https://github.com/fedimint/fedimint/pull/4972/commits/0d6392922bb2fd4170fdeb1316d69acb464998d7#diff-74be73affedba5c5deec399f9ecc2011a75d6c914998fd385604c9b1e013024eR36 I did similiar change and turned `FM_BITCOIN_RPC_KIND` etc. into `FM_FORCE_BITCOIN_RPC_KIND` and `FM_DEFAULT_BITCOIN_RPC_KIND`
